types/onfido-sdk-ui/index.d.ts

```diff
@@ -0,0 +1,62 @@
+// Type definitions for onfido-sdk-ui 6.0
+// Project: https://github.com/onfido/onfido-sdk-ui#readme
+// Definitions by: Liam Read <https://github.com/LiamDotPro>
+//                 aktary <https://github.com/aktary>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export interface iOnfidoResponse {
+    document_front: {
+        id: string;
+        type: string;
+        side: string;
+    };
+    face: {
+        id: string;
+        variant: string;
+    };
+}
+
+export interface iOnfidoConfig {
+    token?: string;
+    containerId?: string;
+    onComplete?: (data: iOnfidoResponse) => null;
+    onModalRequestClose?: () => null;
+    isModalOpen?: boolean;
+    useModal?: boolean;
+
+    steps?: Array<
+        | string
+        | {
+              type: string;
+              options: {
+                  title?: string;
+                  descriptions?: [string];
```

Comment:
Is there always exactly one description?  If not, you probably want `string[]`.

Reply:
Good spot, I hadn't noticed :+1: